docs/getting-started/rulesets.md

```diff
@@ -120,9 +120,15 @@ Now all the rules in this ruleset will only be applied if the specified format i
 
 Custom formats can be registered via the [JS API](../guides/javascript.md), but the CLI is limited to using the predefined ones.
 
+## Given
+
+The `given` keyword is, besides `then`, the only required property on each rule definition. 
+It can be any valid JSONPath expression or an array of JSONPath expressions.
```

Comment:
@P0lip Nit: From a reader's perspective, it's not very clear if, for the rule to pass, all the expressions have to satisfy the expectation or at least one of them.